ci/edmtool.py

```diff
@@ -121,6 +121,29 @@
 
 doc_ignore = {
     "*/tests",
+    "chaco/barplot.py",
```

Comment:
It'd be good to add a reason here as to why these are ignored. I think we can create an issue - regarding the removal of the stubs - and link that here.